code/mission/missionbriefcommon.cpp

```diff
@@ -899,10 +907,24 @@ void brief_render_fade_outs(float frametime)
 				continue;
 			}
 
-			bm_get_info( fi->fade_anim.first_frame, &w, &h, NULL);
+			float scaled_w, scaled_h;
+
+			bm_get_info( fi->fade_anim.first_frame, &w, &h, nullptr);
 			float screenX = tv.screen.xyw.x;
 			float screenY = tv.screen.xyw.y;
-			gr_unsize_screen_posf( &screenX, &screenY, NULL, NULL, GR_RESIZE_MENU_NO_OFFSET );
+
+			int this_resize = bscreen.resize;
+			if (bscreen.resize == GR_RESIZE_MENU) {
+				this_resize = GR_RESIZE_MENU_NO_OFFSET;
+			}
+			gr_unsize_screen_posf(&screenX, &screenY, nullptr, nullptr, this_resize);
+
+			scaled_w = w * scale_factor;
+			scaled_h = h * scale_factor;
+			bxf = screenX - scaled_w / 2.0f + 0.5f;
+			byf = screenY - scaled_h / 2.0f + 0.5f;
+			bx = fl2i(bxf);
+			by = fl2i(byf);
```

Comment:
Before these appear to be used, the next few lines will overwrite them again. 
As such, is this code either not really needed, or is the reassignment below erroneous?

Reply:
I was working on 2 related PRs and once and I swore I removed the errored ones..